cogs/misc.py

```diff
@@ -420,6 +429,15 @@ async def parse(self, ctx, *, time_and_arg: converters.HumanTime(other=True)):
                            f'Other argument: {time_and_arg.other}\n'
                            f'Delta: {(time - pendulum.instance(ctx.message.created_at)).in_words()}')
 
+    @commands.command()
+    async def someone(self, ctx, *, text)
+        """
+        Repeats the message but with a random member's name after :someone
+        """
```

Comment:
`text` argument is not documented. Document it in the docstring using the Google's docstring style guidelines.  
Also, it seems like that `text` is a member. Use the `Member` converter if that's the case, otherwise specify the type in the docstring

Reply:
The only argument passed is a message to be echoed after the random member name (which has now been documented).